core/client/ui/toolboxes/entity-editor.js

```diff
@@ -5,12 +5,24 @@ const entityDepthRange = { min: -1, max: 1000 };
 
 const closeInterface = () => Session.set('selectedEntityId', undefined);
 const selectedEntity = () => Entities.findOne(Session.get('selectedEntityId'));
+const linkedEntity = () => Entities.findOne({ _id: selectedEntity()?.entityId });
 const targets = () => {
   const activeEntityId = Session.get('selectedEntityId');
   return Entities.find({ _id: { $ne: activeEntityId }, actionType: entityActionType.none }).fetch();
 };
 
 Template.entityEditor.events({
+  'click .back_btn'() {
+    closeInterface();
+  },
+  'click .switch_anim'() {
+    const entity = selectedEntity();
+    if (typeof entity.entityId === 'string') {
```

Comment:
I don't think it's useful to test id types everywhere, there are backend checks to do that

Reply:
ok I replace by `if (entity.entityId)`